CHANGES.md

```diff
@@ -1,5 +1,65 @@
 # Changelog
 
+## 6.1 (2023-11-03)
+[View detailed changelog](https://github.com/spinalcordtoolbox/spinalcordtoolbox/compare/6.0...6.1)
+
+**FEATURE**
+ - **sct_analyze_lesion**: Add function to output the axial damage ratio + minor improvements. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4207)
+ - **sct_download_data**: Color installed datasets for `sct_download_data -h`. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4257)
+ - **sct_download_data**: Add current `sct_tutorial_data` release as a new course dataset. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4271)
+ - **sct_run_batch**: Add start date information to the logs. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4218)
+
+**ENHANCEMENT**
+ - **sct_compute_compression**: Improvements of CLI output printed by the `sct_compute_compression`. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4175)
+ - **sct_deepseg**: Add checks for empty arrays post-segmentation. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4199)
```

Comment:
One type of PR I struggled with was "improved error handling" PRs. Are they enhancements? Documentation PRs?

Reply:
I put this in "BUG", because I think we can loosely consider "failing on empty arrays post-segmentation" buggy behavior? Kind of? (At the very least, exceptions are semantically related to bugs, so... it kind of works...)

Reply:
I think "BUG" is a fine categorization for better error handling :+1: